frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,139 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
+import { Faculty } from '../types';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
+    faveEat: string;
+    interests: string;
+  }>({ emojis: '', faculty: '', faveEat: '', interests: '' });
+
+  // TODO:
+  // Add appropriate type to event
+  const handleChange = (e: any) => {
+    const { name, value } = e.target;
+    setUserDetails({
+      ...userDetails,
+      [name]: value,
+    });
+  };
+
+  const handleSubmit = async () => {
+    console.log(userDetails);
+    // TODO:
+    // Hit the correct API to successfully store info to the db
+    const res = await fetch('http://localhost:3001/updateUser', {
+      method: 'POST',
+    });
+    await res.json();
+  };
+
+  return (
+    <Background bgcolor={'#fff'}>
+      <Grid
+        item
+        xs={12}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Typography variant={'h5'}>This is...</Typography>
+        <Typography variant={'h2'}>TuesHey</Typography>
+        <Typography variant={'h4'}>
+          Let's get to know you <br /> ✌️😙✌️
+        </Typography>
+      </Grid>
+      <Grid
+        item
+        xs={12}
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-around',
+          alignItems: 'center',
+        }}
+      >
+        <div style={{ width: '25rem' }}>
+          <TextField
+            onChange={handleChange}
+            value={userDetails.emojis}
+            placeholder={'😎 ⚽ 🍔'}
+            name="emojis"
+            label={'Describe yourself in 3 emojis!'}
+            fullWidth
+          />
+        </div>
+        <div style={{ width: '25rem' }}>
+          <FormControl fullWidth>
```

Comment:
We **need** to get the Faculty from the user. We can mark this field as required by setting a `required` property against this FormControl. [Here's the docs](https://mui.com/material-ui/react-select/#other-props) for more info